test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetAddPackageTests.cs

```diff
@@ -945,12 +924,7 @@ public async Task AddPkg_WithCPM_WithPackageReference_NoVersionOverride_NoPackag
                     packageX100,
                     packageX200);
 
-            var propsFile = @$"<Project>
-                                <PropertyGroup>
-                                    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-                                </PropertyGroup>
```

Comment:
Do we need one test to ensure CPM restore works as expected for projects where `ManagePackageVersionsCentrally` property has been configured explicitly?

Reply:
Sounds like a good idea to me.

Reply:
This new test is doing this, `true` is the user explicitly opted in, `false` is they opted out, and `null` is they did not specify so they get the default of enabled since Directory.Packages.props exists.

https://github.com/NuGet/NuGet.Client/pull/5456/files#diff-1faf54421a99e7ac571eb0ca8dd73601dc5d38ee395668da1373fef077488154R10111-R10114